stories/infinite-hits.stories.js

```diff
@@ -68,4 +69,34 @@ storiesOf('InfiniteHits', module)
         })
       );
     })
+  )
+  .add(
+    'with previous button enabled',
+    withHits(
+      ({ search, container, instantsearch }) => {
+        search.addWidget(
+          instantsearch.widgets.infiniteHits({
+            container,
+            showPrevious: true,
+            templates: {
+              item: '{{name}}',
+            },
+          })
+        );
+      },
+      {
+        routing: {
+          stateMapping: {
```

Comment:
We should update this part to make the story usable from the Storybook. We can implement a small memory router that saves the different `routeState`. It would be nice to also logs the value into the [action](https://github.com/algolia/instantsearch.js/blob/577e4ccb33155d0d4678b04505132d39a2918f22/.storybook/decorators.ts#L37-L46) tab (like we do for the rest).